Sources/AloeStackView/AloeStackView.swift

```diff
@@ -551,16 +574,20 @@ open class AloeStackView: UIScrollView {
     if animated {
       cell.alpha = 0
       layoutIfNeeded()
-      UIView.animate(withDuration: 0.3) {
+      UIView.animate(withDuration: 0.3, animations: {
         cell.alpha = 1
+      }) { flag in
+        completion?(flag)
       }
+    } else {
+      completion?(true)
     }
   }
 
-  private func removeCell(_ cell: StackViewCell, animated: Bool) {
+  private func removeCell(_ cell: StackViewCell, animated: Bool, completion: ((Bool) -> Void)? = nil) {
     let aboveCell = cellAbove(cell: cell)
 
-    let completion: (Bool) -> Void = { [weak self] _ in
+    let completion: (Bool) -> Void = { [weak self] flag in
```

Comment:
The name of the handler conflicts with the argument to the method. Perhaps rename the local variable to `completionHandler` to disambiguate. Also the argument to the function might be better called `finished` rather than `flag`.

Reply:
thank you for your feedback!
i applied all feedbacks. could you check one more time?